thunder/torch/__init__.py

```diff
@@ -782,6 +782,17 @@ def reshape(a: TensorLike, /, *shape: int) -> TensorLike:
     return clang.reshape(a, shape)
 
 
+@torchsymbol(torch.unflatten, is_method=True)
+def unflatten(a: TensorLike, /, dim: int, sizes=tuple[int, ...]) -> TensorLike:
```

Comment:
This signature looks pretty nice, but `sizes-tuple[int, ...]` is assigning the type annotation, and the type annotation is overly strict because any `Sequence` is OK, so it should be:

```
sizes: Sequence[int]
```

Reply:
You are right! Didn't know PyTorch supported lists and other sequence as well. Will update :) Thanks!